src/game/game_state.c

```diff
@@ -1430,3 +1430,7 @@ bool is_demoplay(game_state *gs) {
 bool is_twoplayer(game_state *gs) {
     return !is_demoplay(gs) && !is_netplay(gs) && !is_singleplayer(gs);
 }
+
+bool is_rec_playback(game_state *gs) {
+    return strlen(gs->init_flags->rec_file) > 0 && gs->init_flags->playback == 1;
```

Comment:
```suggestion
    return gs->init_flags->rec_file[0] != '\0' && gs->init_flags->playback == 1;
```
If you don't need the string length, don't call strlen.